structure/0072-legal-team.md

```diff
@@ -0,0 +1,85 @@
+# RFC 0072: Legal Team
+
+This document has been written according to the process outlined in [RFC 0006](https://github.com/QuiltMC/rfcs/blob/main/structure/0006-governance.md#teams).
+
+# Summary
+
+The Legal Team is a standalone team in charge of handling legal matters to the best of its ability. This team will not require its members to be lawyers 
+or have any form of training, although this would be preferred. 
```

Comment:
On a point of formatting: some lines within this document have trailing spaces. I believe those should be removed.